File: 1-two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        list=[]
        for i in range(len(nums)-1):
            for j in range(i,len(nums)-1):
                if nums[i]+nums[j+1]==target:
                    list.append(i)
                    list.append(j+1)
                    return(list)
        
        

File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {

        HashMap<Integer, Integer> map=new HashMap<>();
        int[] temp= new int[2];
        for(int i=0;i<nums.length;i++){
            int need=target-nums[i];
            if(map.containsKey(need)){
                temp[0]=map.get(need);
                temp[1]=i;
                return temp;
            }
            map.put(nums[i],i);
        }
        return temp;
    }
}